src/server/services/wms/qgsmaprendererjobproxy.cpp

```diff
@@ -27,15 +27,11 @@ namespace QgsWms
   QgsMapRendererJobProxy::QgsMapRendererJobProxy(
     bool parallelRendering
     , int maxThreads
-#ifdef HAVE_SERVER_PYTHON_PLUGINS
```

Comment:
@elpaso checked that build is ok with plugins disabled. Removed  HAVE_SERVER_PYTHON_PLUGINS defines from parameters and fixed other defines accordingly.